tencentcloud/resource_tc_kubernetes_cluster.go

```diff
@@ -231,6 +231,27 @@ func TkeCvmCreateInfo() map[string]*schema.Schema {
 				return
 			},
 		},
+		// payment
+		"instance_charge_type": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ForceNew:     true,
+			Default:      CVM_CHARGE_TYPE_POSTPAID,
+			ValidateFunc: validateAllowedStringValue(TKE_WORKER_CHARGE_TYPE),
+			Description:  "The charge type of instance. Valid values are `PREPAID` and `POSTPAID_BY_HOUR`, The default is `POSTPAID_BY_HOUR`.",
+		},
+		"instance_charge_type_prepaid_period": {
+			Type:         schema.TypeInt,
+			Optional:     true,
+			ValidateFunc: validateAllowedIntValue(CVM_PREPAID_PERIOD),
```

Comment:
should ForceNew and has a defautl value.

Reply:
Will fix, and please check the same code in `resource_tc_instance`.